.travis.yml

```diff
@@ -1,34 +0,0 @@
-language: groovy
```

Comment:
Looks like I was wrong and the Travis build can fail randomly for no apparent reason. This is still a problem but less than was led to believe. I'll reactivate the build on all my branches.